lib/Wikia/src/Service/User/Attributes/AttributeKeyValueService.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Wikia\Service\User\Attributes;
+
+use Wikia\Logger\Loggable;
+use Wikia\Persistence\User\Attributes\AttributePersistence;
+use Wikia\Util\WikiaProfiler;
+use Wikia\Service\User\Attributes;
+
+class AttributeKeyValueService implements AttributeService {
```

Comment:
I added it in the API for the service, but I don't think it's a very common operation. From what I can tell the User class doesn't even expose a way to do it, you can get and update attributes, but not delete.

That being said, it is a functionality we expose in the API so we should probably support it in the client as well. I'll update.